front/admin/cli.ts

```diff
@@ -324,8 +324,8 @@ const dataSource = async (command: string, args: parseArgs.ParsedArgs) => {
       if (!args.wId) {
         throw new Error("Missing --wId argument");
       }
-      if (!args.provider) {
-        throw new Error("Missing --provider argument");
+      if (!args.connectorId) {
```

Comment:
I think it's easier to keep `provider` here, as getting `connectorId` requires some SQL on end user's side

Reply:
Yes ideally we keep provider for operations for now :+1: